docs/README.md

```diff
@@ -327,6 +333,59 @@ the application with the selected profile.
 ```shell
 export AWS_PROFILE=other_profile
 ```
+#### Azure Provider Prerequisites
+
+This step is optional, in the case you want to use the Azure Provider. 
+
+> **Note**:
+>
+> Booster is free to use, but notice that the resources deployed to your cloud provider
+> might generate some expenses.
+>
+> In Azure, when you're not eligible for the free tier, resources are charged on-demand. Deploying a
+> Booster project and sending a few commands and queries should cost just a few cents.
+>
+> In any case, make sure to un-deploy your application with the command `boost nuke -e production`
+> if you're not planning to keep using it.
+
+Now it is a good time to create that Microsoft Azure account. You can do so from [the Microsoft SingUp page](https://signup.live.com/). Then you can go to your Azure account in this [link](https://azure.microsoft.com).
+
+Once you have created the Azure account, you need to install the [Azure CLI](https://docs.microsoft.com/es-es/cli/azure/install-azure-cli) in your machine, by following the instructions of the Azure CLI documentation.
+
+Apart from that, you need to install [jq](https://stedolan.github.io/jq/download/) in your system.
+
+The final step consists of signing in in your Azure account from the Azure CLI:
+
+```shell
+az login
+```
+
+You can check the different login options in this [reference document](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli). 
+
+Once you are logged in from the CLI, you can deploy applications with Booster.
+
+> You can see more details in [Providers configuration](#providers-configuration) section.
+
+#### Kubernetes Provider Prerequisites
+
+This step is optional, in the case you want to use the Kubernetes provider. 
+
+The main requirement is having a Kubernetes Cluster already configured. This provider has been succesfully tested in EKS (Amazon Elastic Kubernetes Service), AKS (Azure Kubernetes Service) and GKE (Google Kubernetes Engine). Since Kubernetes is a standard, this provider can also work in other Kubernetes clusters, including on-premises configurations.
+
+> **Note**:
+>
+> Booster is free to use, but notice that the Kubernetes cluster resources used for the deployment
+> might generate some expenses.
+
+On the other hand, it is also required to install **kubectl** and [Helm](https://helm.sh). 
```

Comment:
I would rephrase the beginning `On the other hand`  because we are not contrasting with something like `Additionally, it is also ....`

Regarding Helm we need Helm version >=3 I think that we can call it Helm3 (not really sure about this name) or just mention the minimum version

Reply:
Makes sense. I'll add the version